apps/src/block_utils.js

```diff
@@ -1074,16 +1075,17 @@ exports.createJsWrapperBlockCreator = function (
       helpUrl: getHelpUrl(docFunc), // optional param
       init: function () {
         // Styles should be used over hard-coded colors in Google Blockly blocks
-        if (style && this.setStyle) {
-          this.setStyle(style);
-        } else if (color) {
+        if (color) {
```

Comment:
nit: I'd accept the argument that it's less clear and I'm not sure `chosenColor` is the right variable name, but could also do something like...
```
const chosenColor = color || (!returnType ? DEFAULT_COLOR : null);
if (chosenColor) {
    Blockly.cdoUtils.setHSV(this, ...chosenColor);
}
```

Reply:
I think we can't quite do that because setting the color to `null` (or any other negative value) would override the color that was previously set by the `returnType`. If there's a `returnType` at all, it's dangerous to call `.setHSV`. 

We could probably do something like this:
```
if (!returnType) {
  const blockColor = color || DEFAULT_COLOR;
  Blockly.cdoUtils.setHSV(this, ...blockColor);
}
```
but then also slightly changes the behavior. This is probably an edge case to be fair. Right now a levelbuilder could specify a block to have both a `returnType` and a `color`, and the color would take precedence. That seems worth leaving in to me.